app/templates/ratemypdf_stats.html

```diff
@@ -236,6 +277,161 @@ <h2 class="pdf-section-heading">Suggestions</h2>
+              <div class="accordion-body">
+                <ul>
+                {% for sentence in stats.get("passive sentences", []) %}
+                <li>
+                  {{ highlight_text(sentence[0], sentence[1])  | safe }}
+                </li>
+                {% endfor %}
+                </ul>
+              </div>
+            </div>
+          </div>
+          {% endif %}
+          {% if stats.get("difficult words", []) %}
+          <div class="accordion-item">
+            <h2 class="accordion-header" id="heading-difficult-words">
+              <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-difficult-words" aria-expanded="false" aria-controls="collapse-difficult-words">
+                Consider replacing difficult words
+              </button>
+            </h2>
+            <div id="collapse-difficult-words" class="accordion-collapse collapse" aria-labelledby="heading-difficult-words" data-bs-parent="#suggestionsAccordion">
+              <div class="accordion-body">
+                We refer to [plainlanguage.gov](https://www.plainlanguage.gov/guidelines/words/)'s list of common
+                complex words and simple replacements. Apply judgement and read the words in context to make sure the replacements
+                still make sense.
+                <ul>
+                {% for word in stats.get("difficult words", []) %}
+                <li>
+                  {{ word  }}
+                </li>
+                {% endfor %}
+                </ul>
+              </div>
+            </div>
+          </div>
+          {% endif %}
+          {% if stats.get("plain language suggestions") %}
+          <div class="accordion-item">
+            <h2 class="accordion-header" id="heading-plain-lang-suggestions">
+              <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-plain-lang-suggestions" aria-expanded="false" aria-controls="collapse-plain-lang-suggestions">
+                Consider substituting simpler words and phrases
+              </button>
+            </h2>
+            <div id="collapse-plain-lang-suggestions" class="accordion-collapse collapse" aria-labelledby="heading-plain-lang-suggestions" data-bs-parent="#suggestionsAccordion">
+              <div class="accordion-body">
+                <table class="table">
+                  <thead>
+                    <tr>
+                      <th scope="col">Original sentence</th>
+                      <th scope="col">Revised sentence</th>
+                    </tr>                
+                {% for suggestion in stats.get("plain language suggestions", []) %}
+                <tr>
+                  <td>
+                    {{ suggestion[0] | safe }}
+                  </td>
+                  <td>{{ highlight_text(suggestion[1], suggestion[2]) | safe  }}</td>
```

Comment:
Pretty minor nit, but the plain language suggestions sometimes include the original word in the recommendations, which I'm not a huge fan of. [Exhibit maps onto show and exhibit](https://github.com/SuffolkLITLab/FormFyxer/pull/101/files#diff-07092260481383b9e7562a70e69647b8c1a7779d0f117196bc210e58fe34de77R91) in one I found. Tied into the [bigger discussion about what to do with weird substitution results](https://teams.microsoft.com/l/message/19:910fed53a41c433091cce11066c866ae@thread.tacv2/1682083018110?tenantId=78733fa9-540e-4eb8-bf29-73c4eeb63412&groupId=eaa9bd9d-cf39-4686-8f30-e55aa9d98c75&parentMessageId=1682083018110&teamName=Legal%20Innovation%20and%20Technology%20Lab&channelName=assembly-line-backend-development&createdTime=1682083018110&allowXTenantAccess=false).

Reply:
I added exhibit manually because sometimes it is a noun in the legal context ("exhibit 1") and replacing it is incorrect. Yeah, detecting part of speech might solve this, or we could remove it as too likely to cause a false positive. Or maybe add a percentage guess that we're correct.

Going to leave this in for now though.